README.md

```diff
@@ -510,9 +516,13 @@ To compress PNGs and SVGs in the current directory and its subdirectories, two c
 
 # Releasing
 
-1. Create a pull request. Get it reviewed and approved.
-1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate.
-1. Merge the pull request.
+1. Create a pull request with prep for the new release. This should contain changes to release notes and anything else that needs to be done.
+1. Get it reviewed and approved, and merge the pull request.
+1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate. This will:
+  - Update versions in `package.json` and `package-lock.json`
+  - Commit those changes locally and tag that commit with the new version
+  - "Compile" and publish the changes (**based on what is in your local working tree**, so make sure you're cleanly on master with no local changes) to npm
+1. `git push && git push --tags` to push the npm generated commit and tag up to origin
```

Comment:
you can't push this change directly to master because of the repository permisisons. if you run npm version patch after the pr is merged, you'll need to create a new pr with the version change. that is why the previous instructions required you to run npm version after sign-off but before merging the pull request.